platformio.ini

```diff
@@ -27,22 +27,13 @@
 test_dir = tests/
 
 [env]
-; https://docs.platformio.org/en/feature-platformio-30/projectconf.html
-src_filter = +<*> -<.git/> -<.svn/> -<example/> -<examples/> -<arduino-ide/SIGNALDuino/> -<_micro-api/libraries/ArduinoJson/test/> -<_micro-api/libraries/ArduinoJson/third-party/> -<_micro-api/libraries/fastdelegate/examples/>-<test/> -<tests/> -<_micro-api/libraries/ArduinoJson/fuzzing/>
-lib_extra_dirs = 
-    src/_micro-api/libraries/
-
-; general - for all devices, using ${env.***}
-
-; Unix, example
-;debug_port = /dev/ttyUSB0
-;monitor_port = /dev/ttyUSB0  ; only devices without WLAN
-;upload_port = /dev/ttyUSB0
-
-; Windows, example
-debug_port = COM6
-monitor_port = COM6          ; only devices without WLAN
-upload_port = COM6
+build_src_filter = +<*> -<.git/> -<.svn/> -<example/> -<examples/> -<arduino-ide/SIGNALDuino/> -<_micro-api/libraries/ArduinoJson/test/> -<_micro-api/libraries/ArduinoJson/third-party/> -<_micro-api/libraries/fastdelegate/examples/>-<test/> -<tests/> -<_micro-api/libraries/ArduinoJson/fuzzing/>
+lib_extra_dirs = src/_micro-api/libraries/
+#debug_port = COM6
+#monitor_port = COM6
+#upload_port = COM6
+upload_speed = 921600
+extra_scripts = pre:extra_script.py
```

Comment:
Plattformio mäkelt hier: option 'extra_scripts' in section 'env' already exists
Nach Auskommentieren lässt es sich compilieren.